### ISSUE TITLE
helm-update-dep Error - prometheus-operator

### ISSUE THREAD

**Issue by getshellz** [saferwall/saferwall#255]
Hi 

am getting this error when trying to build with kind on Ubuntu 18.04 (local), when following this https://github.com/saferwall/saferwall/blob/master/docs/DEPLOYING-DEV.md 

are you able to advise?

```
sudo make helm-update-dep
cd  /home/lab-admin/saferwall/deployments \
	&& helm dependency update saferwall
Getting updates for unmanaged Helm repositories...
...Successfully got an update from the "https://kubernetes-charts.storage.googleapis.com/" chart repository
...Successfully got an update from the "https://kubernetes-charts.storage.googleapis.com/" chart repository
...Successfully got an update from the "https://kubernetes-charts.storage.googleapis.com/" chart repository
...Successfully got an update from the "https://kubernetes-charts.storage.googleapis.com/" chart repository
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "couchbase" chart repository
...Successfully got an update from the "ingress-nginx" chart repository
...Successfully got an update from the "elastic" chart repository
...Successfully got an update from the "jetstack" chart repository
...Successfully got an update from the "stable" chart repository
Update Complete. ⎈Happy Helming!⎈
Saving 10 charts
Downloading minio from repo https://kubernetes-charts.storage.googleapis.com/
Downloading couchbase-operator from repo https://couchbase-partners.github.io/helm-charts/
Downloading efs-provisioner from repo https://kubernetes-charts.storage.googleapis.com/
Downloading nfs-server-provisioner from repo https://kubernetes-charts.storage.googleapis.com/
Downloading cert-manager from repo https://charts.jetstack.io
Downloading ingress-nginx from repo https://kubernetes.github.io/ingress-nginx/
Downloading filebeat from repo https://helm.elastic.co
Downloading elasticsearch from repo https://helm.elastic.co
Downloading kibana from repo https://helm.elastic.co
Downloading prometheus-operator from repo https://kubernetes-charts.storage.googleapis.com/
Save error occurred:  could not download https://charts.helm.sh/stable/prometheus-operator-9.3.2.tgz: failed to fetch https://charts.helm.sh/stable/prometheus-operator-9.3.2.tgz : 404 Not Found
Deleting newly downloaded charts, restoring pre-update state
Error: could not download https://charts.helm.sh/stable/prometheus-operator-9.3.2.tgz: failed to fetch https://charts.helm.sh/stable/prometheus-operator-9.3.2.tgz : 404 Not Found
build/mk/helm.mk:35: recipe for target 'helm-update-dep' failed
make: *** [helm-update-dep] Error 1
```

**Comment by LordNoteworthy** [comment]
Right ! This comes comes their end, I start facing the same issue as well, I just reproduced it now. Not really sure why this is happening, I know that they are deprecating the stable repo now.

Let me see if I can move away from https://github.com/helm/charts.

**Comment by LordNoteworthy** [comment]
Should be fixed in: 65e713942bb4d1a32a3c5f5c207efbbc9fbfb9e1.
